packages/web-react/src/components/Pagination/README.md

```diff
@@ -132,6 +144,16 @@ This component extends the `PaginationButtonLink` component.
 | -------------------- | -------- | ------- | -------- | ------------------------------- |
 | `accessibilityLabel` | `string` | `Next`  | ✕        | Accessibility label of the link |
 
+## UncontrolledPagination props
+
+| Name                 | Type                           | Default | Required | Description                                |
+| -------------------- | ------------------------------ | ------- | -------- | ------------------------------------------ |
+| `accessibilityLabel` | `string`                       | ✕       | ✕        | Accessibility label of the link            |
+| `defaultPage`        | `number`                       | `1`     | ✕        | Page number of default page                |
+| `chapterSize`        | `number`                       | `5`     | ✕        | Number of displayed pages between chevrons |
```

Comment:
how about page count or something like that? I am not sure if the keyword chapter is the right one.